### PR TITLE
Config changes for QA Fire

### PR DESCRIPTION
**Pull request by micapam**
- Added env var to allow deploys after first one to succeed
- Removed additional memory config as it wasn't working

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ qafire:
   buildpack: https://github.com/cloudfoundry-incubator/multi-buildpack
   command: npm install -g gulp && npm run build && bundle exec rake assets:precompile && bin/cf-start.sh
```
Comment (elisechant) [inline manifest.yml]: can change to just `npm install build`. no need to install anything global

**Comment by elisechant** [comment]
love this PR title

**Comment by micapam** [comment]
Ah sorry @elisechant I didn't mean this to be reviewed... is just a dummy PR to test QA Fire stuff.

File: manifest.yml (hunk 1 above)
Comment (micapam) [inline manifest.yml]: Happy to change this, but can we do it outside this PR @elisechant? As the command hasn't changed as part of the PR.

**Comment by micapam** [comment]
This is no longer a dummy PR, BTW ... it's ready for the merge to conform with changes as per this QA Fire PR: https://github.com/AusDTO/qa-fire/pull/111

**Pull request merged**